_data/mentors.yml

```diff
@@ -1166,7 +1166,7 @@
 
 - name: Rasim Sen
   disabled: false
-  sort: 200
+  sort: 100
```

Comment:
Have you checked with him if it is not available @mads6879 ?

Reply:
His availability in the file was only for July, and he's not on the spreadsheet for mentors who responded with more availability